lib/templates/application/app.rb

```diff
@@ -149,6 +165,22 @@ def sign_in_as_admin!
     sign_in(:admin, Admin.first) unless Admin.all.empty?
   end
 
+  def seo(name)
+    begin
+      is_a_resource? ? resource.setting(name, nil, role: 'Admin') : resource_class.setting(name, nil, role: 'Admin')
+    rescue
```

Comment:
Discussed with Bill and we've decided to leave this for now as the handling of SEO may change soon.